### PR TITLE
Create 2020-01-05-carpentries-dc-workshop-iran.md

### PR DESCRIPTION
**Pull request by kmomar**

### CODE REVIEW

**Comment by kmomar** [comment]
referencing Issue #968

**Comment by kmomar** [comment]
@ElnazAmanzadeh : Please review this Pull Request of your Blog Post and let me know if there are any changes before publishing on January 5

File: _posts/2021/01/2020-01-05-carpentries-dc-workshop-iran.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+authors: ["Elnaz Amanzadeh "]
+teaser: "This post covers the instructor's experiences teaching an online Data Carpentry workshop in Iran"
+title: "Running an Online Data Carpentry Workshop in Iran"
+date: 2020-1-05
```
Comment (zkamvar) [inline _posts/2021/01/2020-01-05-carpentries-dc-workshop-iran.md]:
```suggestion
date: 2021-01-05
```

Comment (kmomar) [reply _posts/2021/01/2020-01-05-carpentries-dc-workshop-iran.md]: Ah! Thank you!

**Comment by ElnazAmanzadeh** [comment]
> 
> 
> _No description provided._
**Omar Khan**
Sorry for delay. I was n the process of my PhD final and thesis works. I reviewed the changes. It seems okay for me and no changes is needed. Thanks.

**Pull request merged**